cmd/soroban-cli/src/invoke.rs

```diff
@@ -432,12 +442,20 @@ impl Cmd {
             }
         }
 
-        snapshot::commit(state.1, ledger_info, &storage.map, &self.ledger_file).map_err(|e| {
+        snapshot::commit(state.1, &ledger_info, &storage.map, &self.ledger_file).map_err(|e| {
             Error::CannotCommitLedgerFile {
                 filepath: self.ledger_file.clone(),
                 error: e,
             }
         })?;
+
+        snapshot::commit_events(&events.0, &ledger_info, &self.events_file).map_err(|e| {
+            Error::CannotCommitLedgerFile {
+                filepath: self.events_file.clone(),
+                error: e,
+            }
+        })?;
```

Comment:
This needs to create the file if it doesn't exist, otherwise it will error:
```
error: committing file .soroban/events.json: No such file or directory (os error 2)
```

Reply:
And needs to mkdir the dir in case the file is different from the ledger.json.

Reply:
This is done by the snapshot, no? e.g. this part of `snapshot::commit_events`:

https://github.com/stellar/soroban-tools/blob/77a2120cf34028cc0084a6be95205a18abcddb63/cmd/soroban-cli/src/snapshot.rs#L156-L165